packages/terra-core-docs/src/terra-dev-site/test/button/TabIndex.test.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Button from 'terra-button';
```

Comment:
Given that we now have this test file, we should add a test step in https://github.com/cerner/terra-core/blob/main/packages/terra-button/tests/wdio/button-spec.js to actually use this test example. Otherwise, this isn't being tested except just having the test example.